_posts/2025-03-21-release.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Release 2025.12.1
+author: Priya
+date: 2025-03-21
+category: Jekyll
+layout: post
+---
+### Product Enhancement :
+* Data Settings: Added a dropdown in the Data Settings modal to allow users to select a quote character during data ingestion.
```

Comment:
Let's not include this in release notes. It is made as a workaround for Juan - not even an alpha feature. We will keep this only until ingestion algo is sorted.